hooks/build

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+docker build --build-arg=VERSION=$DOCKER_TAG -t $IMAGE_NAME .
```

Comment:
should this be `--build-arg VERSION=$DOCKER_TAG`

https://docs.docker.com/engine/reference/commandline/build/